### PR TITLE
Search page refactor

### PR DESCRIPTION
**Pull request by broskoski**
search

### CODE REVIEW

File: react/components/SearchContents/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                  {contents.map(cell => ({
+                    Image: <Cell.Konnectable key={`${cell.__typename}_${cell.id}`} konnectable={cell} />,
+                    Attachment: <Cell.Konnectable key={`${cell.__typename}_${cell.id}`} konnectable={cell} />,
+                    Text: <Cell.Konnectable key={`${cell.__typename}_${cell.id}`} konnectable={cell} />,
+                    Link: <Cell.Konnectable key={`${cell.__typename}_${cell.id}`} konnectable={cell} />,
+                    Embed: <Cell.Konnectable key={`${cell.__typename}_${cell.id}`} konnectable={cell} />,
+                    Channel: <Cell.Konnectable key={`${cell.__typename}_${cell.id}`} konnectable={cell} />,
+                    User: <Cell.Identifiable key={`${cell.__typename}_${cell.id}`} identifiable={cell} />,
+                    Group: <Cell.Identifiable key={`${cell.__typename}_${cell.id}`} identifiable={cell} />,
+                  }[cell.__typename]))}
```
Comment (dzucconi) [inline react/components/SearchContents/index.js]: So you probably want to wrap these in functions, like https://github.com/aredotna/ervell/blob/master/react/components/Cell/components/Konnectable/index.js#L120-L128

This is rendering all of these at once, currently

File: react/components/SearchMetadata/components/SearchMetadataModeView/index.js [hunk 2]
```diff
@@ -1,52 +1,73 @@
...
+      <Pocket title="View">
+        <CookieLinkUnlessCurrent
+          to={`/search/${search}/all`}
+          isActive={this.isViewActive('all')}
+          prefix="Search"
+          name="view"
+          value="all"
+        >
+          All
+        </CookieLinkUnlessCurrent>
```
Comment (dzucconi) [inline react/components/SearchMetadata/components/SearchMetadataModeView/index.js]: What was the problem, ultimately?

Comment (broskoski) [reply react/components/SearchMetadata/components/SearchMetadataModeView/index.js]: An errant router hoc wrapped on the outer component

File: react/components/SearchContents/index.js (hunk 1 above)
Comment (broskoski) [reply react/components/SearchContents/index.js]: Shall do

**Pull request merged**